srt-tokio/src/tokio/socket.rs

```diff
@@ -266,8 +267,19 @@ where
                                     // neither--this exists just to keep the connection alive
                                     KeepAlive => {}
                                     Srt(s) => {
-                                        dbg!(s);
-                                        // unimplemented!("{:?}", s);
+                                        match s {
```

Comment:
`socket` should not care about a low-level detail like this--this code here just decides which state machine to send various packets. So how this should look is here, we should just send these packets to both sender and receiver (probably just add to the Shutdown case, ie `Shutdown | Srt(_)`, then in the receiver handle_packet add the call to rekey. 

The background here is srt-tokio is just supposed to be a thin layer ontop of the base state machines, not have any actual protocol logic. I know that there are some protocol details here, but they should be moved and abstracted at some point.